### ISSUE TITLE
[CODE] CLRNet

### ISSUE THREAD

**Issue by alessandrodicosola** [alessandrodicosola/FellowshipResearch#32]
PAPER: #12 
- [x] Setup code on Google Colab
- [x] Build nms on Google Colab
- [x] Setup code on server
- [x] Build nms on server
- [x] Test code

| **Code** | ResNet18 | ResNet34 | ResNet101 |
|----------|----------|----------|-----------|
| CULane   |    :heavy_check_mark:     |     :heavy_check_mark:     |    :heavy_check_mark:      |
| TuSimple |      :heavy_check_mark:      |    Not available      |      Not available     |
| LLAMAS   |          |          |           |

## Hyperparameters in testing
conf_threshold: threshold softmax "lineness"
nms_threshold
nms_topk
